lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -230,16 +232,15 @@ export class FormFieldModel extends FormWidgetModel {
             this.leftLabels = form.json.leftLabels || false;
         }
 
-        const emptyOption = Array.isArray(this.options) ? this.options.find(({ id }) => id === 'empty') : undefined;
-        if (this.hasEmptyValue === undefined) {
-            this.hasEmptyValue = json?.hasEmptyValue ?? !!emptyOption;
-        }
+        this.updateForm();
```

Comment:
was this already inside the get value? please test this code very well it could break several things

Reply:
Need more info, don't get it